contrib/google.golang.org/grpc/tags.go

```diff
@@ -5,8 +5,29 @@
 
 package grpc
 
+import "google.golang.org/grpc"
+
 // Tags used for gRPC
 const (
-	tagMethod = "grpc.method"
-	tagCode   = "grpc.code"
+	tagMethod     = "grpc.method"
```

Comment:
Does this value still show up properly in the UI when there's both a value and a group with the same name?

Reply:
Good question. I'll double check.

Reply:
I just checked and it does not appear correctly.
I believe that `tagMethod` should actually be `tagMethodName` per our docs, and following the example of the python integration:
https://github.com/DataDog/dd-trace-py/blob/73b0ba15248a0b8050b1f6cfca814468be384bf9/ddtrace/contrib/grpc/constants.py#L9

Is this a breaking change, or can I change this tag?

Reply:
It means even adding `grpc.method.kind` is a potentially breaking change.
So we'll need to discuss it a bit and decide the right way(s) to solve it.
In some ways, it's a general problem, we just happened to hit it very specifically.

Reply:
I would say to change the old one to `grpc.method.name`. The current name is inconsistent with our internal RFC and the likelihood of this affecting anyone is small. If we encounter any problems, we can release a bug fix to revert and call the kind one `grpc.method_kind` instead. But I would not be a big fan of the latter.